src/sagemaker/model.py

```diff
@@ -302,6 +302,16 @@ def transformer(self, instance_count, instance_type, strategy=None, assemble_wit
                            env=env, tags=tags, base_transform_job_name=self.name,
                            volume_kms_key=volume_kms_key, sagemaker_session=self.sagemaker_session)
 
+    def delete_model(self):
+        """Delete an Amazon SageMaker ``Model``.
+
+        Raises: ValueError if model is not deployed yet.
```

Comment:
this should be formatted:

```
Raises:
    ValueError: if the model has not been created
```

also I think you mean "created" not "deployed" - only once a SageMaker model is created can it then be deployed

Reply:
I updated the doc strings here and below as well as test's expected error message.